cdi-core/src/main/java/com/linkedin/cdi/factory/http/HttpRequestMethod.java

```diff
@@ -55,6 +55,27 @@
  */
 
 public enum HttpRequestMethod {
+  /**
+   * Note: This works when the URI template has all the variable parameters.
+   * If there are any additional parameters, then URIBuilder encodes all parameters while building.
+   */
+  GET_XE("GET_XE") {
```

Comment:
We may need to update the documentations on supported methods as well?
https://github.com/linkedin/data-integration-library/blob/616886debf6c107218764d466540db13d53f8552/docs/parameters/ms.http.request.method.md

Reply:
Added a related comment. This is a hidden feature. Hence, the cryptic name as well